NEWS_abbr.md

```diff
@@ -0,0 +1,287 @@
+# NEWS
+
+## polars 1.0.0
+
+This is a completely rewritten new version of the polars R package. It improves
+the internal structure of the package and catches up with Python Polars' API.
+At the time of writing, this version of R Polars matches Python Polars 1.31.0.
+
+Therefore it contains many breaking changes compared to the previous R Polars
+implementation. Some of those breaking changes are explained below, but many
+others are due to modifications of function names, argument names, or argument
+positions. There are too many to list here, so you should refer to the [Python
+Polars API docs](https://docs.pola.rs/api/python/dev/reference/index.html).
+
+Installation instructions for the rewritten version are now identical to the
+instructions for the old version. The old version (referred to as `polars0` in
+the docs) can be installed from R-universe with:
+
+```r
+Sys.setenv(NOT_CRAN = "true")
+install.packages("polars0", repos = "https://rpolars.r-universe.dev")
+```
+
+### Breaking changes
+
+* The class names of `polars` objects have changed:
+  - `RPolarsLazyFrame` -> `polars_lazy_frame`
+  - `RPolarsDataFrame` -> `polars_data_frame`
+  - `RPolarsSeries` -> `polars_series`
+  - `RPolarsExpr` -> `polars_expr`
+
+* Conversion from unknown classes to Polars objects now fail. Developers can
+  specify how those objects should be handled by `polars` by creating a method
+  for `as_polars_series.<my-object>`.
+
+  ```r
+  ### OLD
+  a <- 1
+  class(a) <- "foo"
+  as_polars_series(a)
+  #> polars Series: shape: (1,)
+  #> Series: '' [f64]
+  #> [
+  #>         1.0
+  #> ]
+  ```
+
+  ```r
+  ### NEW
+  a <- 1
+  class(a) <- "foo"
+  as_polars_series(a)
+  #> Error:
+  #> a <foo> object can't be converted to a polars Series.
+  #> Run `rlang::last_trace()` to see where the error occurred.
+  ```
+
+* Conversion from `polars` to R types has been revamped: `$to_r()`, `$to_list()`
+  and `$to_data_frame()` no longer exist. Instead, you must use `as.data.frame()`,
+  `as.list()` and `as.vector()`.
+
+  If finer control is needed (for instance to specify how integer-64 or date
+  columns should be handled), use either `$to_r_vector()` or set options for the
+  entire session with `polars_options()`.
+
+* In general, `polars` now uses dots (`...`) in two scenarios:
+
+  1. to pass an unlimited number of inputs (for instance in `select()`, `cast()`,
+     or `group_by()`), using [dynamic-dots](https://rlang.r-lib.org/reference/dyn-dots.html).
+
+    For example, if you used to pass a vector of column names or a list of
+    expressions, you now need to expand it with `!!!`:
```

Comment:
(At least on GitHub) listings are not considered contiguous.
Please adjust for blank lines and line breaks.

Reply:
Indeed, this part was missing a white space in indentation